src/main/resources/application.yaml

```
@@ -186,3 +190,12 @@ springdoc:
     enabled: true
     operationsSorter: method
   writer-with-order-by-keys: true
+
+professional:
+  refdata:
+    pageSize: ${PROFESSIONAL_REFDATA_PAGE_SIZE:100}
+  role:
+    mapping:
+      scheduling:
+        enabled: ${PROFESSIONAL_SCHEDULING_ENABLED:true}
```

```suggestion
        enabled: ${PROFESSIONAL_SCHEDULING_ENABLED:false}
```
maybe we should disable scheduling for now